src/Microsoft.ML.Core/Data/ProgressReporter.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.ML.Runtime.Data
     public static class ProgressReporting
     {
         /// <summary>
-        /// The progress channel for <see cref="TlcEnvironment"/>.
+        /// The progress channel for <see cref="ConsoleEnvironment"/>.
```

Comment:
>ConsoleEnvironment [](start = 48, length = 18)

Is it really only TlcEnvironment specific or can be used in any HostEnvironmentBase descendant?

Reply:
I think it could be used elsewhere, but I'm not sure, and I would rather not try now.

---
In reply to: [217869313](https://github.com/dotnet/machinelearning/pull/923#discussion_r217869313) [](ancestors = 217869313)